docs/uperf.md

```diff
@@ -71,34 +83,46 @@ metadata:
   name: example-benchmark
   namespace: ripsaw
 spec:
+  user: test_user # user is a key that points to user triggering ripsaw, useful to search results in ES
```

Comment:
If the 'user' key is an arbitrary value for us to search/match on in Elastic, I'd prefer to use a different key name that makes this more clear. The 'user' key to me implies a username/userid that I need to get right in order for things to work.